jme3-core/src/main/java/com/jme3/material/Material.java

```diff
@@ -890,18 +890,51 @@ private boolean isBO(final VarType type) {
         return type == VarType.BufferObject;
     }
 
-    private void updateRenderState(RenderManager renderManager, Renderer renderer, TechniqueDef techniqueDef) {
+    private void updateRenderState(Geometry geometry, RenderManager renderManager, Renderer renderer, TechniqueDef techniqueDef) {
         if (renderManager.getForcedRenderState() != null) {
-            renderer.applyRenderState(renderManager.getForcedRenderState());
+            if (!renderManager.getForcedRenderState().isFaceCullFlippable() || !isNormalsBackward(geometry.getWorldScale())) {
+                renderer.applyRenderState(renderManager.getForcedRenderState());
+            }
+            else {
+                RenderState flipped = renderManager.getForcedRenderState().clone();

```

Comment:
This creates a new object every time updateRenderState is called, that is every frame.
We need a different solution to avoid overloading the garbage collector.